### ISSUE TITLE
MSSQL tedious "options.encrypt" warning

### ISSUE THREAD

**Issue by exp-anoop** [sequelize/sequelize#9574]
Hi,
When I tried to connect mssql database it shows the following warning message.

```
The default value for `options.encrypt` will change from `false` to `true`. Please pass `false` explicitly if you want to retain current behavior.
```
When I check the sequelize documentation I could not find an option to set "encrypt" to false. 

__Dialect:__ mssql
__Sequelize version:__ 4.37.10
__tedious version:__ 2.6.1

I think we need add on option to set "encrypt" parameter in sequalize connect.

**Comment by sushantdhiman** [comment]
Fixed by #9588

By default Sequelize will set encrypt to false. You can easily override this by `dialectOptions`

```js
var sequelize = new Sequelize('my - connection - string', {
  dialect: 'mssql',
  dialectOptions: {
    encrypt: true
  }
});
```

**Comment by vinaybhawsar** [comment]
Above solution didin't work for me to override this by dialectOptions.
I tried bellow way and it worked for me

`var sequelize = new Sequelize('my - connection - string', {
  dialect: 'mssql',
  dialectOptions: {
        options: {
            encrypt: true
        }
  }
});`
